shell/src/main/java/org/apache/kafka/shell/MetadataNodeManager.java

```diff
@@ -94,38 +94,14 @@ public void handleCommit(BatchReader<ApiMessageAndVersion> reader) {
         }
 
         @Override
-        public void handleCommits(long lastOffset, List<ApiMessage> messages) {
-            appendEvent("handleCommits", () -> {
-                log.debug("handleCommits " + messages + " at offset " + lastOffset);
-                DirectoryNode dir = data.root.mkdirs("metadataQuorum");
-                dir.create("offset").setContents(String.valueOf(lastOffset));
-                for (ApiMessage message : messages) {
-                    handleMessage(message);
-                }
-            }, null);
-        }
-
-        @Override
-        public void handleNewLeader(MetaLogLeader leader) {
+        public void handleLeaderChange(LeaderAndEpoch leader) {
             appendEvent("handleNewLeader", () -> {
```

Comment:
This is probably outside the scope of this PR but it looks like `queue` is never read.